topics/bazis.qmd

```diff
@@ -1,6 +1,7 @@
 ---
-title: BAZIS HPC
+title: High-Performance Computing facilities
```

Comment:
```suggestion
title: High-Performance Computing Facilities
```

Ensuring it follows our emerging standard to capitalize the titles (see also https://capitalizemytitle.com/)